packages/scene-composer/src/interfaces/sceneViewer.ts

```diff
@@ -9,6 +11,11 @@ export interface DracoDecoderConfig {
   path?: string;
 }
 
+export interface PanelType {
+  direction: typeof Direction.Left | typeof Direction.Right;
+  panels: Record<string, JSX.Element>;
+}
```

Comment:
this file is only meant for the public interfaces for SceneViewer component. the panelType is an internal component interface that should be in the other types file in the components folder